gui/package.json

```diff
@@ -68,7 +69,7 @@
     "chai-spies": "^1.0.0",
     "cross-env": "^5.1.3",
     "electron": "^11.2.3",
-    "electron-builder": "^22.8.0",
+    "electron-builder": "^22.10.4",
```

Comment:
Could you push the changes to `package-lock.json` as well?

Reply:
Forgive my ignorance (my experience with node.js is quite limited), but how do I do that exactly? If I run `npm install` I get way more changes in `package-lock.json` than what I changed in `package.json` (and I also get some changes in `package.json` btw).

Reply:
Which version of `npm` are you using? It could be that your using 6.x? I’ll try it out myself in a moment and see whats going on.

Reply:
7.5.0

Reply:
Yeah there seems to be quite a lot of changes due to `electron-builder` having a few dependencies. Just leave it as it currently is and I'll fix it after this PR has been merged.